src/ResourceManagement/Sql/SqlManagement/Generated/Models/EncryptionProtectorProperties.cs

```diff
@@ -41,15 +41,15 @@ public string ServerKeyName
             set { this._serverKeyName = value; }
         }
         
-        private string _type;
+        private string _serverKeyType;
         
         /// <summary>
         /// Optional. The Transparent Data Encryption Encryption Protector Type
         /// </summary>
-        public string Type
+        public string ServerKeyType
```

Comment:
@shahabhijeet I bumped the nugget package version with commit https://github.com/Azure/azure-sdk-for-net/pull/2834/commits/6f8b3356997125c2833488e26aae24783f0670d9